operator/cmd/mesh/operator-init.go

```diff
@@ -82,6 +82,13 @@ func operatorInit(args *rootArgs, oiArgs *operatorInitArgs, l clog.Logger) {
 	if err != nil {
 		l.LogAndFatal(err)
 	}
+
+	// Create operatorNamespace if it does not exist.
+	l.LogAndPrintf("Creating operator namespace %s if it is not created", oiArgs.common.operatorNamespace)
```

Comment:
Can we only log if its actually created? It is a bit confusing for users otherwise?

Reply:
Since `createNamespace` doesn't differentiate cases when the namespace already exists from cases when it is created, I've added a function `namespaceExists` in `operator/cmd/mesh/operator-common.go` in order to be able to print this log only when it is actually created with [this block](https://github.com/kabooboo/istio/commit/06db78751def0d02a336a66eb9bb046cf0d1c531#diff-9459faf88adc550cc9e905154a8e1f37R87-R95).